lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -491,35 +492,40 @@ $LCMCredentials= New-Object System.Management.Automation.PSCredential ($LCMUserN
 
 ### Task05 - Create Azure Resources
 
-Following script will simply create Resource Group and ARC Gateway (optional).
+Following script will simply create Resource Group and Arc Gateway (optional).
```

Comment:
Typo for Arc (repeats through this file).

Reply:
Ok